src/gui/mainwindow.cpp

```diff
@@ -1944,6 +1944,8 @@ void MainWindow::updatePowerManagementState()
         return torrent->isMoving();
     });
     m_pwr->setActivityState(inhibitSuspend);
+
+    m_preventTimer->start(PREVENT_SUSPEND_INTERVAL);
```

Comment:
Can't understand why did you move it here.

Reply:
I assume the iteration above is costly so the timer should reset to avoid excessive updates.

Reply:
Then it would also make sense to turn the timer to "single shot" mode, wouldn't it?

Reply:
Done.
Also changed the PR title.

Previously, the commit message was misleading. The power management update is working as intended and this PR only slightly revise it.